### PR TITLE
doc: cephfs: update with atime not supporting explaination

### PR DESCRIPTION
**Pull request by lxbsz**
Signed-off-by: Xiubo Li <xiubli@redhat.com>

More detail please see the discussion in ceph patchwork: https://patchwork.kernel.org/project/ceph-devel/list/?series=633625

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

File: doc/cephfs/posix.rst [hunk 1]
```diff
@@ -40,6 +40,11 @@ POSIX semantics for various reasons:
   code.  The name of this hidden directory can be changed at mount
   time with ``-o snapdirname=.somethingelse`` (Linux) or the config
   option ``client_snapdir`` (libcephfs, ceph-fuse).
+- CephFS doesn't maintain the atime for now. Some existing applications
+  maybe using the atime and will move files off to a secondary storage
+  if they haven’t been accessed for a certain time period. But it's
+  allowed to update the atime via the SETATTR for some use cases, for
+  exmaple backup.
```
Comment (gregsfortytwo) [inline doc/cephfs/posix.rst:47]: Content looks good, but maybe:

> CephFS does not currently maintain the ``atime`` field. Most applications do not care, though this impacts some backup and data tiering applications that can move unused data to a secondary storage system. You may be able to workaround this for some use cases, as CephFS does support setting ``atime`` via the ``setattr`` operation.

Comment (lxbsz) [reply doc/cephfs/posix.rst]: Looks much better. Done. Thanks @gregsfortytwo

File: doc/cephfs/posix.rst [hunk 2]
```diff
@@ -40,6 +40,11 @@ POSIX semantics for various reasons:
   code.  The name of this hidden directory can be changed at mount
   time with ``-o snapdirname=.somethingelse`` (Linux) or the config
   option ``client_snapdir`` (libcephfs, ceph-fuse).
+- CephFS does not currently maintain the atime field. Most applications
+  do not care, though this impacts some backup and data tiering
+  applications that can move unused data to a secondary storage system.
+  You may be able to workaround this for some use cases, as CephFS does
+  support setting atime via the setattr operation.
```
Comment (anthonyeleven) [inline doc/cephfs/posix.rst:47]: Suggest ```atime``` and ```setattr``` in double backticks (```)
LGTM otherwise

Comment (lxbsz) [reply doc/cephfs/posix.rst]: Done. Thanks @anthonyeleven

**Review by anthonyeleven — APPROVED** [review]

**Pull request merged**